source/globalCommands.py

```diff
@@ -1,9 +1,8 @@
 # -*- coding: UTF-8 -*-

```

Comment:
Feel free to remove this line

Reply:
It must be present because there is a letter Ł in one of contributors name.

Reply:
Now  the encoding declaration can be removed, since the encoding is specified when invoking Gettext.